pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -737,7 +737,6 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 
     // merge the data returned by progress api into the data returned by the section students api
     $scope.mergeProgress = function() {
-      // TODO
```

Comment:
It was left over from when I renamed course -> script in a bunch of places, and should not have ever been committed.